File: app/src/main/java/com/project/thisappistryingtomakeyoubetter/model/Task.java
package com.project.thisappistryingtomakeyoubetter.model;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.Ignore;
import androidx.room.PrimaryKey;

import java.util.Date;

@Entity
public class Task {
    @PrimaryKey(autoGenerate = true)
    int id = 0;
    @ColumnInfo(name = "title")
    String title;
    @ColumnInfo(name = "description")
    String description;
    @ColumnInfo(name = "date")
    Date date;
    @ColumnInfo(name = "finish", defaultValue = "false")
    boolean finish;

    @Ignore
    public Task(String title, String description){
        this.title = title;
        this.description = description;
    }

    public Task(String title, String description, Date date){
        this.title= title;
        this.description = description;
        this.date = date;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public boolean isFinish() {
        return finish;
    }

    public void setFinish(boolean finish) {
        this.finish = finish;
    }
}

File: app/src/main/java/com/project/thisappistryingtomakeyoubetter/model/Task.kt
package com.project.thisappistryingtomakeyoubetter.model

import androidx.room.PrimaryKey
import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.Ignore
import java.util.*
import kotlin.collections.ArrayList

@Entity
class Task {
    @PrimaryKey(autoGenerate = true)
    var id = 0

    @ColumnInfo(name = "title")
    var title: String?

    @ColumnInfo(name = "description")
    var description: String?

    @ColumnInfo(name = "date")
    var date: Date? = null

    @ColumnInfo(name = "finish", defaultValue = "false")
    var isFinish = false

    @Ignore
    var labels: MutableList<Label> = ArrayList()

    @Ignore
    constructor(title: String, description: String) {
        this.title = title
        this.description = description
    }

    constructor(title: String, description: String, date: Date?) {
        this.title = title
        this.description = description
        this.date = date
    }

    override fun toString(): String {
        return "Task $title" +
                " With label Label $labels}"
    }
}